client/client/modules/render/core/track/track.js

```diff
@@ -406,8 +406,9 @@ export class Track extends BaseTrack {
             this.state.coverageScaleFrom = state.data.from;
             this.state.coverageScaleTo = state.data.to;
             this.state.coverageScaleMode = scaleModes.manualScaleMode;
+            this.state.coverageDisplayMode =  this.state.coverageDisplayMode || displayModes.defaultDisplayMode;
```

Comment:
This will add `coverageDisplayMode` setting to the track state, even if it doesn't make sense for this track (like for `FEATURE_COUNTS` tracks).
We should **override** this method (`coverageScaleSettingsChange`) for tracks that have specific logic (like checking `featureCountsDisplayMode` instead of `coverageDisplayMode` for `FEATURE_COUNTS`)

Reply:
@rodichenko Added condition in `coverageScaleSettingsChanged` method for `FEATURE_COUNTS` track format  to set right `isLogScale` value